CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.34.0]
+### Changed
+- fix: update berachain  `chainid` ([#205](https://github.com/MetaMask/ppom-validator/pull/205))
```

Comment:
Nice, PPOM team included in the [ppom_version.json](https://static.metafi.codefi.network/api/v1/confirmations/ppom/ppom_version.json) the new `0x138d4`.